next.config.js

```diff
@@ -1,12 +1,7 @@
 const withPlugins = require('next-compose-plugins')
 const withCSS = require('@zeit/next-css')
-const withTM = require('next-transpile-modules')([
-    '@adobe/react-spectrum',
-    '@spectrum-icons/.*',
-    '@react-spectrum/.*'
-])
 
-module.exports = withPlugins([withCSS, withTM], {
+module.exports = withPlugins([withCSS], {
```

Comment:
Is `withCSS` still required with the way that CSS is working in MUI? I believe we only added it for the sake of Spectrum.

Reply:
oh interesting! I'll check that out. Was wondering what "zeit" was other than a pretty decent centrist newspaper in germany

Reply:
Nice you're right, next supports css out of the box and this was unneeded, and giving warnings in the console. 2 birds, 1 stone.

Reply:
Great idea for the branch, thank you!